README.md

```diff
@@ -29,14 +29,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+for each doesnt create a new array while map does
 2. What is the difference between a function and a method?
-
+a method is a function inside a function, a regular function is usually not inside 
```

Comment:
a function inside an object, as a property on that object. a function inside a function is just a nested function